packages/core-extensions/src/moonbase/ui/config/index.tsx

```diff
@@ -108,15 +109,39 @@ export default (require: WebpackRequireType) => {
   }
 
   return function ConfigPage() {
+    const TitleBarClasses = spacepack.findByCode("iconWrapper:", "children:")[0]
+      .exports;
     return (
       <>
+        <div
+          className={`${TitleBarClasses.children} ${Margins.marginBottom20}`}
+        >
+          <Text
+            className={TitleBarClasses.titleWrapper}
+            variant="heading-lg/semibold"
+            tag="h2"
+          >
+            Config
+          </Text>
+        </div>
```

Comment:
Also applies to the other page, these should not have the title bar classes, that was only for when the title bar also had the tabs (which I am very sad to see gone, but oh well), you should revert them to how the Moonbase header originally worked.